src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,61 @@
+package tasks;
+import java.io.Serializable;
+import java.time.LocalDate;
+/**
+ * Representation of a task
+ * recorded by the chatbot.
+ * @author Alvis Ng (supermii2)
+ */
+public abstract class Task implements Serializable {
+    /** Name of task */
+    protected String taskName;
+    /** One letter abbreviation for the task. */
+    protected String oneLetterAbbrev;
```

Comment:
This is a good idea! I like that the variable name is intuitive. The comment helps too!

Reply:
Thank you!